codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-watch: true
+watch: false
```

Comment:
hmm, when working with graphql queries though, I found this useful. If my graphql is incorrect it will refresh with error, until i save it correctly. I am ok with manually re-running it though